docs/hadoop-catalog-with-gcs.md

```diff
@@ -487,6 +487,7 @@ GVFS Java client:
 
 ```java
 Configuration conf = new Configuration();
+conf.setBoolean("fs.gravitino.enableCredentialVending", true);
```

Comment:
Please modify the description about how to enable credential vending.

Reply:
added